clients/src/main/java/org/apache/kafka/common/requests/BasePartitionState.java

```diff
@@ -16,38 +16,24 @@
  */
 package org.apache.kafka.common.requests;
 
-import org.apache.kafka.common.utils.Utils;
-
 import java.util.List;
 
```

Comment:
@becketqin Sure. I added this comment. Is this OK?

`This class contains the common fields shared between LeaderAndIsrRequest.PartitionState and UpdateMetadataRequest.PartitionState`.